README.md

```diff
@@ -1,264 +1,483 @@
+5. If you are working with the Maven CLI, you can manually install the `.jar` into your Maven
+   repository by running the following (replacing `$VERSION` with the appropriate version
+   of the API):
+   ```sh-session
+   $ mvn org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file \
+       -Dfile=/path/to/api/repo/target/conjur-api-$VERSION.jar
+   ```
+   or
+   ```sh-session
+   $ mvn org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file \
+       -Dfile=/path/to/api/repo/target/conjur-api-$VERSION-with-dependencies.jar
+   ```
+   or
+   ```sh-session
+   $ mvn install:install-file -Dfile=/path/to/api/repo/target/conjur-api-$VERSION-with-dependencies.jar \
+       -DgroupId=net.conjur.api \
+       -DartifactId=conjur-api \
+       -Dversion=$VERSION \
+       -Dpackaging=jar
+   ```
 
 ## Configuration
-Once the setup steps have been successfully run, we will now define the environment variables needed to make the connection between the new app and Conjur. The best way to do so is by setting environment variables. 
-In Conjur (both OSS and DAP), environment variables are mapped to configuration variables by prepending `CONJUR_` to the all-caps name of the
-configuration variable. For example, `appliance_url` is `CONJUR_APPLIANCE_URL`, `account` is `CONJUR_ACCOUNT` etc.  
-
-_NOTE:_ For ways to set credentials (for `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY` environment variables ONLY) in the app instead of environment variables, see the [Authorization Examples](#authorization-examples) section below.
-
-The following environment variables need to be included in the apps runtime environment in order use the Conjur API:
-- `CONJUR_ACCOUNT`
-- `CONJUR_AUTHN_LOGIN`
-- `CONJUR_AUTHN_API_KEY`
-- `CONJUR_APPLIANCE_URL`
-- `CONJUR_AUTHN_URL`
-
-`CONJUR_ACCOUNT` - account specified during Conjur setup
-
-`CONJUR_APPLIANCE_URL` - Conjur HTTPS endpoint (OSS/DAP)
 
-`CONJUR_AUTHN_LOGIN` - user/host identity
-
-`CONJUR_AUTHN_API_KEY` - user/host API key
+Once the setup steps have been successfully run, we will now define the variables needed
+to make the connection between the new app and Conjur. You can do this by setting
+[environment variables](#environment-variables), [system properties](#system-properties),
+or some combination of both.
+
+_NOTE:_ System properties will override enviroment values when both are defined for a
+variable.
+
+### Environment Variables
+
+In Conjur (both OSS and DAP), environment variables are mapped to configuration variables
+by prepending `CONJUR_` to the all-caps name of the configuration variable. For example,
+`appliance_url` is `CONJUR_APPLIANCE_URL`, `account` is `CONJUR_ACCOUNT` etc.
+
+_NOTE:_ For ways to set credentials (for `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY`
+environment variables ONLY) in the app instead of environment variables, see the
+[Authorization Examples](#authorization-examples) and/or [system properties](#system-properties)
+section below.
```

Comment:
I'm not sure that I totally understand this note.  Can these credentials only be set via env variables?  Or do the linked headers only apply to setting them as env variables?  Could you make it a little clearer?